### ISSUE TITLE
Performing multiple LHS estimations fails on panel data objects

### ISSUE THREAD

**Issue by nreigl** [lrberge/fixest#158]
While performing multiple LHS estimation works on dataframe objects, it seems to fail on panel data objects. 

First, performing estimations on multiple LHS dependent variables works as expected: 

```R
library(fixest)

data(base_did)

base_did$y1 <- rnorm(length(base_did)) # create second dependent variable

res_multi <- feols(c(y,y1) ~ x1, base_did)

res_multi


Standard-errors: Standard 
Dep. var.: y
            Estimate Std. Error t value  Pr(>|t|)    
(Intercept)  1.98880   0.149155  13.333 < 2.2e-16 ***
x1           0.98311   0.050119  19.615 < 2.2e-16 ***
---
Dep. var.: y1
            Estimate Std. Error  t value Pr(>|t|) 
(Intercept) 0.028412   0.032279 0.880191 0.378952 
x1          0.002765   0.010847 0.254919 0.798834 
```
I can either convert the dataframe to panel data object or supply the panel id as argument in the model formulae in order to, for example, work with leads and/ or lags. 

However, instead of regressing *y* and *y1* individually on the explanatory variables, `feols` seems to be perform one regression using the sum of *y* and *y1* as dependent variable.  

```R
res_multi <- feols(c(y,y1) ~ l(x1, 0:1), base_did, panel.id = ~id+period)

res_multi

OLS estimation, Dep. Var.: c(y + y1)
Observations: 972 
Standard-errors: Standard 
            Estimate Std. Error   t value  Pr(>|t|)    
(Intercept) 2.247600   0.160479 14.005000 < 2.2e-16 ***
x1          0.994880   0.054073 18.399000 < 2.2e-16 ***
l(x1, 1)    0.028249   0.054943  0.514145  0.607268    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1
RMSE: 4.9943   Adj. R2: 0.257377

```

Note that I have only tried to estimate regressions with multiple manipulations on the RHS with one *stepwise* function on a panel data structured object but it seems to work.  

```R
res_multi <- feols(y ~ l(x1, 0:1) | sw0(post), base_did, panel.id = ~id+period)

res_multi

Standard-errors: mixed (use summary() with arg. 'se' or 'cluster' to harmonize them) 
Fixed-effects: 1
            Estimate Std. Error   t value  Pr(>|t|)    
(Intercept) 2.234800   0.157744 14.167000 < 2.2e-16 ***
x1          0.994800   0.053151 18.716000 < 2.2e-16 ***
l(x1, 1)    0.040962   0.054006  0.758473  0.448353    
---
Fixed-effects: post
         Estimate Std. Error    t value Pr(>|t|)    
x1       1.012200   0.001128 897.700000 0.000709 ***
l(x1, 1) 0.042398   0.051286   0.826693 0.560219  
```

**Comment by lrberge** [comment]
Thanks, fixed by a1353a7c137ff0b2938238bc9af6747c929928e1.

**Comment by nreigl** [comment]
Thank you for the super fast fix!
